arcaflow_plugin_kill_pod.py

```diff
@@ -205,12 +206,15 @@ def kill_pods(
                 new_watch_pods: typing.List[Pod] = []
                 for p in watch_pods:
                     try:
-                        read_pod = core_v1.read_namespaced_pod(p.name, p.namespace)
+                        read_pod = core_v1.read_namespaced_pod(p.name, p.namespace)                        
                         if read_pod.metadata.name != p.name:
                             return "error", PodErrorOutput(
                                 "Error retrieveing pod {}".format(p.name)
                             )
-                        new_watch_pods.append(p)
+                        if read_pod.status.phase=='Running':
```

Comment:
I dont think we need to see if the status is running here. We just want to validate that the pod was killed. The wait for pods portion of the code will validate that the pod and container are running Dont think we need to do extra waiting in this step